packages/shared/src/components/ShareOptionsMenu.tsx

```diff
@@ -77,6 +86,41 @@ export default function ShareOptionsMenu({
     },
   ];
 
+  if (hasSquadAccess) {
+    if (!squads?.length) {
+      shareOptions.push({
+        icon: <MenuIcon Icon={SquadIcon} />,
+        text: 'Post to new squad',
+        action: () =>
+          openModal({
+            type: LazyModal.NewSquad,
+            props: { post },
+          }),
+      });
+    }
+
+    squads?.map(
+      (squad) =>
+        squad.active &&
+        shareOptions.push({
+          icon: squad.image ? (
+            <SquadImage className="mr-2.5 w-5 h-5 text-2xl" {...squad} />
+          ) : (
+            <MenuIcon Icon={DefaultSquadIcon} />
+          ),
+          text: squad.name,
```

Comment:
I think it should be `Share to ${squad.name}` but I'm not sure about the exact copy

Reply:
Updated according to the design
<img width="251" alt="SCR-20230131-ltf" src="https://user-images.githubusercontent.com/1488164/215762982-b3ff6c63-5690-49ea-a1b9-6eae75631885.png">

Reply:
Doubled checked with product and we can change it to `Share to "squad_name"`

Reply:
Updated to `Share to ...`